package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@ts-common/json",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Should you really bump the version here, there isn't any change to the actual package?

Reply:
Actually, yes. Code generation is updated to ES2019 and some comments.

Reply:
Right make sense